### ISSUE TITLE
We should have two generator question paths

### ISSUE THREAD

**Issue by Scotchester** [cfpb/generator-cf#81]
If we started the generator off with the question `Is this going to be a Consumer Financial Protection Bureau project? (y/n)`, we will enable ourselves to

a. Have smarter default values for both CFPB and non-CFPB project metadata, and
b. Include things like our licensed font `@font-face` rules if appropriate, or not.

Having the ability to do (b) would clear up some of the confusion we encountered in #79.

**Comment by contolini** [comment]
We now [check](https://github.com/cfpb/generator-cf/blob/bbbe4eb5f3a67685d071593625811da5b7bb0f3d/app/index.js#L114) to see if `cfpb.gov` is in the user's email address and modify the templates appropriately.

**Comment by Scotchester** [comment]
I still want to make the default metadata smarter depending on the path.

**Comment by contolini** [comment]
What do you have in mind? Auto-complete everything right off the bat if you're a CFPB employee?

**Comment by Scotchester** [comment]
Yeah. A number of things (author stuff, license, maybe others I'm forgetting) don't need to be asked at all if you're a CFPB employee.

**Comment by jimmynotjim** [comment]
We could also install the CFPB specific theming based on email.

**Comment by Scotchester** [comment]
Actually, Chris already partially implemented it that way. I think I'd prefer it be more explicit, though. There's a possibility where someone with a cfpb.gov email address might not want to generate with our theme.

**Comment by contolini** [comment]
To clarify the changes I made:

1. Check if [`cfpb.gov` is in the email address](https://github.com/cfpb/generator-cf/blob/master/app/index.js#L114).
1. If it is, [uncomment licensed fonts](https://github.com/cfpb/generator-cf/blob/3801b13aa5c032864cbd4c87e2cf78f8d2c94421/app/templates/src/static/css/main.less#L17).
